sympy/vector/coordsysrect.py

```diff
@@ -1,19 +1,45 @@
+import types
+from sympy.simplify import simplify
 from sympy.core.basic import Basic
 from sympy.vector.scalar import BaseScalar
-from sympy import eye, trigsimp, ImmutableMatrix as Matrix, Symbol
+from sympy import eye, trigsimp, ImmutableMatrix as Matrix, Symbol, symbols, sqrt, Lambda, Dummy
 from sympy.core.compatibility import string_types, range
 from sympy.core.cache import cacheit
 from sympy.vector.orienters import (Orienter, AxisOrienter, BodyOrienter,
                                     SpaceOrienter, QuaternionOrienter)
-import sympy.vector
 
 
-class CoordSysCartesian(Basic):
+def build_instance_from_lambda(fun=lambda x, y, z: (x, y, z)):
```

Comment:
> depreciation

Are you sure that's the right spelling?
http://english.stackexchange.com/questions/45295/why-is-there-confusion-between-depreciated-and-deprecated

When I hear _depreciation_, I think of a lowering in prices, not of the process of rendering things obsolete.